addons/pos_restaurant/static/src/overrides/models/pos_store.js

```diff
@@ -260,12 +299,8 @@ patch(PosStore.prototype, {
 
         return {
             paidOrdersNotSent,
-            orderToCreate: orderToCreate.filter(
-                (o) => context.table_ids.includes(o.table_id?.id) && !this.tableSyncing
-            ),
-            orderToUpdate: orderToUpdate.filter(
-                (o) => context.table_ids.includes(o.table_id?.id) && !this.tableSyncing
-            ),
+            orderToCreate: orderToCreate.filter((o) => context.table_ids.includes(o.table_id?.id)),
```

Comment:
are you sure you can remove the `!this.tableSyncing` ? Last time I asked you you weren't sure